test/index.test.js

```diff
@@ -2843,16 +2846,16 @@ jobs:
 
     describe('_addPrComment', () => {
         const scmUri = 'github.com:111:branchName';
-        const comment = 'this was a great PR';
-        const jobName = 'main';
+        const comments = [{ text: 'this was a great PR', keyWord: 'foo' }];
```

Comment:
I think it's one word, so can drop camelcase
```suggestion
        const comments = [{ text: 'this was a great PR', keyword: 'foo' }];
```